src/components/Footer.jsx

```diff
@@ -1,11 +1,21 @@
-import React,{useState} from "react";
+import React,{useState,useEffect} from "react";
+import CookieDeclaration from "../components/CookieDeclaration";
 
 const Footer = (props) => {
   const [show,setShow] = useState(false);
+
+  useEffect(() => {
+    var src= "https://consent.cookiebot.com/54349014-3934-489b-a905-584bc83272b2/cd.js"
```

Comment:
There is a constants file, store this over there. and from the constants file take the value from env. something like `REACT_APP_CONSET_COOKIBOT`;
Look at .env file you will understand;